scripts/update_location_summaries.ts

```diff
@@ -79,6 +86,10 @@ async function buildSummaries(
     summaries[countyProjections.fips] = countyProjections.summary;
   }
 
+  for (const projection of allMetroProjections) {
```

Comment:
nit: I'm anal about using `projections` when referring to instances of `Projections` since `Projection` is a different class (another thing we really should clean up. They can/should be merged now!)

Reply:
ahh makes sense